.circleci/config.yml

```diff
@@ -61,7 +61,6 @@ workflows:
           filters:
             branches:
               ignore:
-                - master
```

Comment:
I'm assuming this is so we can get review apps for our "Master -> Prod" PRs, @Niicck could you confirm?

Reply:
Nope. The stage name is now a little bit of a misnomer. I wanted to give us the ability to re-build/re-seed our staging database. I have discovered a problem with my approach. I will change it